### PR TITLE
Add coverage badge

### PR DESCRIPTION
**Pull request by bronzdoc**
Closes https://github.com/rubygems/rubygems.org/issues/1779
Rebased https://github.com/rubygems/rubygems.org/pull/1895 and added code coverage badge

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -19,6 +19,8 @@ tmp/
 tags
 .idea
 chef
+<<<<<<< HEAD
```
Comment (sonalkr132) [inline .gitignore]: Can you please rebase from master?

Comment (bronzdoc) [reply .gitignore]: Done @sonalkr132

File: Gemfile [hunk 2]
```diff
@@ -73,4 +73,9 @@ group :test do
   gem "shoulda"
   gem "selenium-webdriver"
   gem "webdrivers"
+  gem "simplecov", require: false
+end
+
+group :development, :deploy do
```
Comment (sonalkr132) [inline Gemfile]: I don't think you intended to add this.

**Comment by sonalkr132** [comment]
We need to add reporter in our build. [Test Coverage](https://codeclimate.com/github/rubygems/rubygems.org/test_coverage)
https://itnext.io/how-to-setup-codeclimate-and-circleci-2-0-3a85e559bca6#ccd8

**Pull request merged**